SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -67,8 +67,7 @@ print_help() {
     _log "  --repo_type TYPE            Type of repository (git or eos)"
     _log "  --builder BUILDER           Builder to create the environment"
     _log "  --builder_version VERSION   Version of the builder to use (optional)"
-    _log "  --nxcals                    Install NXCALS package and Spark extensions in the environment (optional)"
-    _log "  -h, --help                  Print this help page"
+    _log "  --nxcals                    Install Nxcals extensions in the environment (optional)"    _log "  -h, --help                  Print this help page"
```

Comment:
Why this commit? Wasn't it ok as it was? If --nxcals is provided, we install both NXCALS and the Spark extensions.